src/Functions/currentUser.cpp

```diff
@@ -53,7 +53,30 @@ class FunctionCurrentUser : public IFunction
 
 REGISTER_FUNCTION(CurrentUser)
 {
-    factory.registerFunction<FunctionCurrentUser>();
+    FunctionDocumentation::Description description = R"(
+Returns the name of the current user.
+In case of a distributed query, the name of the user who initiated the query is returned.
+    )";
+    FunctionDocumentation::Syntax syntax = "currentUser()";
+    FunctionDocumentation::Arguments arguments = {};
+    FunctionDocumentation::ReturnedValue returned_value = {"Returns the name of the current user. In distributed queries, the login of the user who initiated the query.", {"String"}};
```

Comment:
```suggestion
    FunctionDocumentation::ReturnedValue returned_value = {"Returns the name of the current user, otherwise the login of the user who initiated the query.", {"String"}};
```